src/Redux/Missions/MissionsAPI.js

```diff
@@ -0,0 +1,34 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+
+import { GET_MISSIONS } from '../Actions';
+
+// API URL
+const URL = 'https://api.spacexdata.com/v3/missions';
+
+const getMissions = createAsyncThunk(GET_MISSIONS, async () => {
+  const response = await fetch(URL);
+  const data = await response.json();
+  console.log(data);
```

Comment:
I think this should be removed Since valuable data can be seen by the end users which is not proper